components/_navbar.html

```diff
@@ -33,6 +34,8 @@
             <!-- Individual RNA-Seq pages go in this list -->
             <li><a href="../03-rnaseq/00-intro-to-rnaseq.html">Introduction</a></li>
             <li><a href="../03-rnaseq/clustering_rnaseq_01_heatmap.html">Clustering and Heatmaps</a></li>
+            <li><a href="../03-rnaseq/dimension-reduction_rnaseq_01_pca.html">Dimension Reduction - PCA</a></li>
```

Comment:
I like your format here but I think there's a section where I may have used a `:`, can you also switch that here? 
We can add this to the CONTRIBUTING.md doc too.